src/app/components/header/header.component.ts

```diff
@@ -1,59 +1,77 @@
 import { Component, OnInit } from '@angular/core';
 import { ChannelService } from './../../services/channel.service';
-import {ApiService} from './../../services/api.service';
-import {PageActionsService} from './../../services/page-actions.service';
+import { ApiService } from './../../services/api.service';
+import { PageActionsService } from './../../services/page-actions.service';
+import { Store } from '@ngrx/store';
+import { Observable } from '../../../../node_modules/rxjs';
+import { User } from '../../models/user';
+import { UserWallet } from '../../models/userWallet';
+import { SET_USER_WALLET } from './../../reducers/userWallet.reducer';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
-  balance: any;
+
+  balance: any = 0;
   updatedAt: any;
+  currentUser: Observable<User>;
+
   constructor(
   	private channelService: ChannelService,
   	private apiService: ApiService,
-    private pageAction: PageActionsService
+    private pageAction: PageActionsService,
+    private store: Store<any>
   	) { }
 
   ngOnInit() {
-  	this.balance = 0;
-  	this.getBalance()
+  	// this.getBalance(); TODO - Remove
```

Comment:
Not a big fan of leaving simple TODO's like this in the code. Best to just remove it.